File: samples/CalculoArea.java
import java.util.Scanner;

public class CalculoArea {
  private static final double PI = 3.14;

  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    int raio = 0;
    boolean maisUm = true;
    System.out.println("+----------------------------+");
    System.out.println("| Cálculo de área do círculo |");
    System.out.println("+----------------------------+");
    System.out.println("");
    while (maisUm) {
      System.out.print("Digite o raio: ");
      raio = scanner.nextInt();
      System.out.println("A área é " + (PI * raio * raio));
      System.out.println("");
      System.out.print("Deseja continuar? (true/false) ");
      maisUm = scanner.nextBoolean();
    }
    scanner.close();
  }

}

File: samples/CalculoArea.kt
/**
 * Programa de exemplo para a linguagem MicroKotlin.
 * Em teoria, este exemplo não deve gerar nenhum erro.
 */

/**
 * Representa o valor da constante pi.
 */
val PI: Double = 3.14;

/**
 * Função principal da aplicação, que será
 * executada assim que o programa iniciar.
 */
fun main() {
  // Inicialização de variáveis.
  var raio: Int = 0;
  var maisUm: Boolean = true;

  // Título
  printLn("+----------------------------+");
  printLn("| Cálculo de área do círculo |");
  printLn("+----------------------------+");
  printLn();

  while (maisUm) {
    print("Digite o raio: ");
    raio = readInt();

    printLn("A área é " + (PI * raio * raio));
    printLn();

    printLn("Deseja continuar? (true/false) ");
    maisUm = readBoolean();
  }
}
